src/Stryker.DataCollector/Stryker.DataCollector/Stryker.DataCollector.csproj

```diff
@@ -2,9 +2,9 @@
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
-    <AssemblyVersion>0.16.1.0</AssemblyVersion>
-    <FileVersion>0.16.1.0</FileVersion>
-    <Version>0.16.1</Version>
+    <AssemblyVersion>0.22.9.0</AssemblyVersion>
+    <FileVersion>0.22.9.0</FileVersion>
+    <Version>0.22.9</Version>
```

Comment:
We should probably add the updating of this version number to the prepare-release.js script we use to bump the version number in the other assemblies.

Reply:
I had a look at the script file, but there is an assumption that each assembly is a project in itself.

Reply:
I'll create an issue to fix this properly 👍👍